src/actor.ts

```diff
@@ -235,17 +235,26 @@ export class EmojiActor {
         this.position.x += this.velocity.x * timeElapsed / this.physics.framerate;
         this.position.y += this.velocity.y * timeElapsed / this.physics.framerate;
 
+        const windowHeight = window.outerHeight ?? document.documentElement.clientHeight;
+        const windowWidth = window.outerWidth ?? document.documentElement.clientWidth;
```

Comment:
Super interesting! Per https://developer.mozilla.org/en-US/docs/Web/API/Window/outerHeight -> https://drafts.csswg.org/cssom-view/#dom-window-outerheight, that value is supposed to _never_ be `NaN`.

> The `outerHeight` attribute must return the height of the client window. If there is no client window this attribute must return zero.

Neither `0` nor `NaN` are considered to be a "falsy" case for the `??` operator so I'm surprised this fix would work for the `NaN` case.

Reply:
oops! I had it working and then cleaned it up with the `??` operator without testing again! just pushed a fix. isNaN won't work in the === 0 case, so changed it to 
`const windowWidth = window.outerWidth ? window.outerWidth : document.documentElement.clientWidth;`

Reply:
Awesome, that makes sense and. We can simplify that further to `window.outerWidth || document.documentElement.clientWidth` then.